### DISCUSSION TITLE
Focus question

### DISCUSSION THREAD

**Discussion by aksld** [alpinejs/alpine#1380] [7. Bugs (archived)]
Hello,

I have a question, I don't know if it's a bug or not.

In the pen : https://codepen.io/aksld/pen/jOBMRbR

My tab system is ok. But after clicking the button stay on state "focus".
This is normally ?

Or maybe I forgot something ?

Thanks

**Comment by SimoTod** [comment]
It's how focus normally works: https://codepen.io/SimoTod/pen/QWpKRad

**Comment by aksld** [comment]
Thanks for help !

So for my need, I have to cancel the focus after the @click ?

**Comment by SimoTod** [comment]
You can add `document.activeElement.blur()` at the end of switch tab but at that point you won't even see it unless you use the keyboard to navigate the page (which may be what you want to achieve).

**Comment by MuzafferDede** [comment]
I think you should use `active` state instead of `focus`

```html
<button class="active:ring-2 focus:outline-none">Test</button>
```
